src/Symfony/Component/Validator/Violation/ConstraintViolationBuilder.php

```diff
@@ -83,6 +83,16 @@ public function disableTranslation(): static
         return $this;
     }
 
+    /**
+     * @return $this
+     */
+    public function setOrDisableTranslationDomain(string|false|null $translationDomain): static
```

Comment:
```suggestion
    public function setTranslationDomain(string|false|null $translationDomain): static
```

Reply:
There is already such a method:

`public function setTranslationDomain(string $translationDomain): static`

We can't pass it `null|false`, but I did not want to put extra checks into each validator to call e. g. `\Symfony\Component\Validator\Violation\ConstraintViolationBuilder::disableTranslation()` in that case. That's why I added a new, all-purpose method.